src/requests/models.py

```diff
@@ -945,7 +945,7 @@ def text(self):
         return content
 
     def json(self, **kwargs):
-        r"""Returns the json-encoded content of a response, if any.
+        r"""Returns the json-decoded dict of a response, if any.
```

Comment:
```suggestion
        r"""Decodes the JSON response body (if any) as a Python object.

           This may return a dictionary, list, etc. depending on what is in the response.
```